### ISSUE TITLE
BUG: React Custom UI `Modal` Unable to display `Code`

### ISSUE THREAD

**Issue by maxming2333** [artf/grapesjs#6308]
### GrapesJS version

- [X] I confirm to use the latest version of GrapesJS

### What browser are you using?

 130.0.6723.117

### Reproducible demo link

https://stackblitz.com/edit/grapesjs-react-custom-ui?file=src%2FApp.tsx

### Describe the bug

**How to reproduce the bug?**
1. open https://grapesjs-react-custom-ui.stackblitz.io/
2. click this button
    <img width="537" alt="image" src="https://github.com/user-attachments/assets/35d33966-60b6-4a88-96c6-0c260d523af7">
4. no code to show in `Modal`
    <img width="1127" alt="image" src="https://github.com/user-attachments/assets/761efaae-5061-4867-8367-832df023a9c8">

**What is the expected behavior?**
show code in `Modal`

**What is the current behavior?**
no code to show in `Modal`

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danstarns** [comment]
Hi @maxming2333, try clicking anywhere inside the highlighted area in the text box, for me it loads the code...


https://github.com/user-attachments/assets/89f61dc6-466a-476a-b437-d0f8ae51c7ef

**Comment by maxming2333** [comment]
> Hi @maxming2333, try clicking anywhere inside the highlighted area in the text box, for me it loads the code...
> 
>  Screen.Recording.2024-11-19.at.23.21.03.mov

Yes, it can indeed be displayed after clicking on the pop-up window area, but it should be displayed as soon as the pop-up window is opened, not after clicking.
